cdap-ui/Gulpfile.js

```diff
@@ -204,15 +205,21 @@ function getEs6Features(isNegate) {
 
 function getEs6Directives(isNegate) {
   var es6directives = [
-    (isNegate ? '!' : '') + './app/directives/dag/**/*.js',
-    (isNegate ? '!' : '') + './app/directives/dag-plus/**/*.js',
-    (isNegate ? '!' : '') + './app/directives/plugin-templates/**/*.js',
-    (isNegate ? '!' : '') + './app/directives/my-global-navbar/*.js',
-    (isNegate ? '!' : '') + './app/directives/datetime-picker/*.js',
-    (isNegate ? '!' : '') + './app/directives/datetime-range/*.js'
+    'dag',
+    'dag-plus',
+    'plugin-templates',
+    'my-global-navbar',
+    'datetime-picker',
+    'datetime-range'
   ];
 
-  return es6directives;
+  var returnVal = [];
+  es6directives.forEach(function (directive) {
```

Comment:
Nothing specific but could be simpler if we do a map here. Something like this,

```
return es6directives.map(function(directive) {
  return (isNegate ? '!' : '') + './app/directives/' + directive + '/**/*.js';
});
```
could avoid ```returnval```